### ISSUE TITLE
adapt for jpg tiles

### ISSUE THREAD

**Issue by abfleishman** [eriqande/dirtybirds#1]
I am interested in using your process on modified images.  I have cropped tiles from Josh's original CR2 images and saved them as jpg at a resolution of ~600x1000 px. This is the max size that can be piped into a workflow using Faster RCNN to find targets and classify them to taxa.  However, this method requires a lot of training data, and we would like to start with your process to identify jpgs that have something interesting in them.  Then once we label them we can see how the neural network does at finding and classifying.
I just forked and cloned your repo, and I am going to read through and see if I can get it to work with my much smaller jpgs. Is there any major issue right off the top of your head that I will need to address to run this with jpgs? Might you be available to chat if I get stuck at some point?  I am hoping to get this going in the next week.

**Comment by eriqande** [comment]
Hey Abram!  

So good to hear from you!  There shouldn't be any issues using smaller images, especially if they are just cropped out of the originals.  If the resolution has changed, then the dimensions of the tiles that dirtybirds create should be changed to accommodate the sizes of birds you might see.  

What I did is super quick and dirty, but is pretty decent at finding interesting targets when there is not a lot of glare.  We can definitely talk next week. Also, I am in Santa Cruz starting next friday for four days if you wanted to meet in person.

Give me a shout any time if you get stuck.

**Comment by abfleishman** [comment]
Thanks Eric!

It would be great to meet in person (I think Matthew would love to chat too). We got one flight of data to play with from Josh and his team, and we were planning to drop it right into our workflow, but because there are so many images without any birds in them we had trouble labeling anything but glare spots, so jump-starting things with a pre-process that can direct us to images that might have a bird or two should be super helpful. 

Thanks for documenting your code enough for me to read through and understand what is going on ;)

**Comment by eriqande** [comment]
That is so great that you and Matthew are working on this with Josh.  

> Thanks for documenting your code enough for me to read through and understand what is going on ;)

Ha! That's the only way **I** can understand it when I come back to it after a few weeks! Looking forward to catching up with you and Matthew.
